features.txt

```diff
@@ -313,6 +313,13 @@ fixes.silverfish_step
 		possible it's not an oversight and that Mojang simply deemed the silverfish step sound too
 		horrible and dummied it out.
 
+fixes.see_items_while_riding
```

Comment:
I don't think this belongs under "fixes", as it's a very intentional visual effect. Tweaks is possibly a better fit.